src/gen_module.sh

```diff
@@ -65,6 +65,37 @@ function get_remote_git_ref() {
     echo "${remote_ref}"
 }
 
+function sphinx_workaround_for_blender29x() {
```

Comment:
As I mentioned in blender.chat, I would like to handle this workaround in RST parser.
If you don't mind, I would like to edit your branch and work at there.

Reply:
As I said, I prefer to have this merged first (it works! + it produces usable 2.90 WHLs) and then work on the upgraded parser in a follow-up commit.
But I understand if you feel uncomfortable with it, so I can also wait with this PR here and rebase when you are done.

Reply:
Thanks for considering about it.
I need some works before this PR, so it is good for me to wait merging this PR.
I will notice you after my fixes are merged to master branch.

Reply:
@grische 

I finished to work all things to merge this PR.
I think this workaround is not required any more.
Could you check with newest master branch?

Thanks for waiting for my works!